sync/class.jetpack-sync-module-users.php

```diff
@@ -114,13 +122,22 @@ public function expand_logout_username( $args, $user_id ) {
 		$user  = get_userdata( $user_id );
 		$user  = $this->sanitize_user( $user );
 		$login = '';
-		if( is_object( $user ) && is_object( $user->data ) ) {
+		if ( is_object( $user ) && is_object( $user->data ) ) {
 			$login = $user->data->user_login;
 		}
 
 		return array( $login, $user );
 	}
 
+	public function deleted_user_handler( $deleted_user_id, $reassigned_user_id = '' ) {
+		do_action( 'jetpack_deleted_user', $deleted_user_id, $reassigned_user_id, is_multisite() );
+	}
+
+	public function user_edited_handler( $user_id ) {
```

Comment:
we should define a naming convention for these hooks  so `deleted_user_handler` and `user_edited_handler` are named similarly